frontend/src/api/auth/authStatus.ts

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+import * as Constants from "../../consts/consts.ts";
+import { UserStatus } from "../../types/authTypes.ts";
+
+export async function getAuthStatus(): Promise<UserStatus> {
+    try {
+        const response = await axios.get<UserStatus>(`${Constants.AUTH_API_URL}/status`);
+        console.log('GET: /auth/status');
+        console.log('Response data:', response.data);
+        return response.data;
+    } catch (error) {
+        console.error('Error getting user status', error);
+        throw error;
+    }
+}
```

Comment:
I think it would be better to return just the permissions. I talked with Samuel, and the user in userStatus isn’t needed on the frontend. It will remain in the API because it might be useful for other apps and isn’t a significant issue. However, we it just causes unnecessary confusion in our app.

Reply:
Will do